script.js

```diff
@@ -1,4 +1,309 @@
+}
+
+// Losing message and add to comScore
+function loseMessage(playerInput, comInput, playerIcon, comIcon) {
+  comScore += 1;
+  return `You Lost! You have chosen ${playerInput} ${playerIcon} while Computer chose ${comInput} ${comIcon}<br> Current Score<br>User:${playerScore}<br>Com:${comScore}<br>Draws:${drawScore}`;
+}
+
+// Draw message and add to drawScore
+function drawMessage(playerInput, comInput, playerIcon, comIcon) {
+  drawScore += 1;
+  return `It's a Draw✏️ You have chosen ${playerInput} ${playerIcon} while Computer chose ${comInput} ${comIcon}<br> Current Score<br>User:${playerScore}<br>Com:${comScore}<br>Draws:${drawScore}`;
+}
+
+// Normal SPS Game function
+function normalGame(user, com) {
+  // If user chose stone and com is scissors, user wins
+  // If user chose paper and com is stone, user wins
+  // If user chose scissors and com is paper, user wins
+
+  if (
+    (user == "stone" && com == "scissors") ||
+    (user == "paper" && com == "stone") ||
+    (user == "scissors" && com == "paper")
+  ) {
+    //Play korean mode if gameMode is korean
+    if (gameMode == "korean") {
+      //Update player winning status and record winning position
+      playerLastWinState = 1;
+      winnerState = 1;
+      return `축하합니다! ${userName} Won! You chose ${user} while Computer chose ${com}! <br>Enter another input to Muk Jji Paa!`;
+    }
+    return winMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+  // If user chose paper and com is scissors, user lose
+  // If user chose scissors and com is stone, user lose
+  // If user chose stone and com is paper, user lose
+  if (
+    (user == "paper" && com == "scissors") ||
+    (user == "scissors" && com == "stone") ||
+    (user == "stone" && com == "paper")
+  ) {
+    //Play korean mode if gameMode is korean
+    if (gameMode == "korean") {
+      //Update computer winning status and record winning position
+      comLastWinState = 1;
+      winnerState = 1;
+      return `죄송합니다! Com Won! You chose ${user} while Computer chose ${com}!<br>Enter another input to fight back!`;
+    }
+    return loseMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+  //else it's a draw
+  else {
+    //Play korean mode if gameMode is korean
+    if (gameMode == "korean") {
+      //Both condition true and draw is obtained, player wins
+      if (playerLastWinState == 1 && winnerState == 1) {
+        //Reset states
+        winnerState = 0;
+        playerLastWinState = 0;
+
+        return (
+          winMessage(user, com, symbolGenerator(user), symbolGenerator(com)) +
+          `<br>${winLossPerc(
+            playerScore,
+            comScore
+          )} Play Again?<br>Current Game Mode:${gameMode}`
+        );
+      }
+      //Both condition true and draw is obtained, computer wins
+      if (comLastWinState == 1 && winnerState == 1) {
+        //Reset states
+        winnerState = 0;
+        comLastWinState = 0;
```

Comment:
Good job factoring in the reset, you could also create a function for that! Since we are calling
```
winnerState = 0;
comLastWinState = 0;
```
a few times